bindings/go/src/_util/translate_fdb_options.go

```diff
@@ -205,16 +201,13 @@ func main() {
 package fdb
 
 import (
-	"bytes"
 	"encoding/binary"
 )
 
-func int64ToBytes(i int64) ([]byte, error) {
-	buf := new(bytes.Buffer)
-	if e := binary.Write(buf, binary.LittleEndian, i); e != nil {
-		return nil, e
-	}
-	return buf.Bytes(), nil
+func int64ToBytes(i int64) []byte {
+	buf := make([]byte, 8)
+	binary.LittleEndian.PutUint64(buf, uint64(i))
```

Comment:
Is it a problem that this takes a signed `int64` and gives it to something that takes an unsigned `uint64`? It shouldn't be, right, as long as a negative `int64` is just converted to the `uint64` with the same bit representation.

Reply:
Yep, it works exactly the same.

Under the hood `binary.Write` just casts the int64 to a uint64 and calls `binary.[endianness].PutUint64`.